views/components/settings/display/theme-config.es

```diff
@@ -55,6 +55,11 @@ const SWITCHES = [
     configName: 'poi.tabarea.grid',
     defaultValue: true,
   },
+  {
+    label: 'Summarize fleet info in overview',
```

Comment:
I think it's make more sense to name the option like 'Use detailed ship info section in main panel' and make the defaultValue to false